tests/test_meta_obj.py

```diff
@@ -0,0 +1,181 @@
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+#
+# Authors:
+#   Valerio Cosentino <valcos@bitergia.com>
+#
+
+import json
+
+import unittest
+
+from archimedes.meta_obj import MetaObj
+
+
+DASHBOARD_REG_ENTRY = {
+    "id": "2e968fe0-b1bb-11e8-8aac-ef7fd4d8cbad",
+    "type": "dashboard",
+    "updated_at": "2019-01-24T13:20:08.902Z",
+    "version": 9,
+    "title": "GitLab Issues"
+}
+
+DASHBOARD_REG_ENTRY_NO_UPDATED_AT = {
+    "id": "2e968fe0-b1bb-11e8-8aac-ef7fd4d8cbad",
+    "type": "dashboard",
+    "version": 9,
+    "title": "GitLab Issues"
+}
+
+DASHBOARD_OBJ = {
+    "attributes": {
+        "description": "GitLab Issues panel by Bitergia",
+        "hits": 0,
+        "kibanaSavedObjectMeta": {
+            "searchSourceJSON": "..."
+        },
+        "optionsJSON": "{\"darkTheme\":false,\"hidePanelTitles\":false,\"useMargins\":true}",
+        "panelsJSON": "...",
+        "timeRestore": "false",
+        "title": "GitLab Issues",
+        "uiStateJSON": "...",
+        "version": 1
+    },
+    "id": "2e968fe0-b1bb-11e8-8aac-ef7fd4d8cbad",
+    "type": "dashboard",
+    "updated_at": "2019-01-24T13:20:08.902Z",
+    "version": 9
+}
+
+DASHBOARD_OBJ_NO_UPDATED_AT = {
+    "attributes": {
+        "description": "GitLab Issues panel by Bitergia",
+        "hits": 0,
+        "kibanaSavedObjectMeta": {
+            "searchSourceJSON": "..."
+        },
+        "optionsJSON": "{\"darkTheme\":false,\"hidePanelTitles\":false,\"useMargins\":true}",
+        "panelsJSON": "...",
+        "timeRestore": "false",
+        "title": "GitLab Issues",
+        "uiStateJSON": "...",
+        "version": 1
+    },
+    "id": "2e968fe0-b1bb-11e8-8aac-ef7fd4d8cbad",
+    "type": "dashboard",
+    "version": 9
+}
+
+
+class TestMetaObj(unittest.TestCase):
+    """MetaObj tests"""
+
+    def test_initialization(self):
+        """Test whether attributes are initialized"""
+
+        obj = MetaObj(_id=DASHBOARD_OBJ['id'],
+                      _title=DASHBOARD_OBJ['attributes']['title'],
+                      _type=DASHBOARD_OBJ['type'],
+                      _version=DASHBOARD_OBJ['version'],
+                      _updated_at=DASHBOARD_OBJ['updated_at'])
+
+        self.assertEqual(obj.id, DASHBOARD_OBJ['id'])
+        self.assertEqual(obj.title, DASHBOARD_OBJ['attributes']['title'])
+        self.assertEqual(obj.type, DASHBOARD_OBJ['type'])
+        self.assertEqual(obj.version, DASHBOARD_OBJ['version'])
+        self.assertEqual(obj.updated_at, DASHBOARD_OBJ['updated_at'])
+
+        obj = MetaObj(_id=DASHBOARD_OBJ_NO_UPDATED_AT['id'],
```

Comment:
This check will work better in a different test method. We ensure both cases work separately.